composer.json

```diff
@@ -15,11 +15,11 @@
         }
     },
     "require": {
-        "php": "^7.1"
+        "php": "8.1.10"
     },
     "require-dev": {
-        "phpunit/phpunit": "^7.1",
-        "squizlabs/php_codesniffer": "^3.3"
+        "phpunit/phpunit": "9.5.25",
+        "squizlabs/php_codesniffer": "3.7.1"
```

Comment:
It's a better practice to add it like `"php": "^8.1"` in order to not specific the minor updates 👍🏻
Same for phpunit and codesniffer.